### PR TITLE
Add Jawg.Lagoon layer

### PR DESCRIPTION
**Pull request by clementgayvallet**
Hello,

I added a new tile layer `jawg-lagoon` for the Jawg provider.

### CODE REVIEW

File: leaflet-providers.js [hunk 1]
```diff
@@ -365,22 +365,22 @@
 			}
 		},
 		Jawg: {
-			url: 'https://{s}.tile.jawg.io/{variant}/{z}/{x}/{y}{r}.png?access-token={accessToken}',
+			url: 'https://tile.jawg.io/{variant}/{z}/{x}/{y}{r}.png?access-token={accessToken}',
 			options: {
 				attribution:
-					'<a href="http://jawg.io" title="Tiles Courtesy of Jawg Maps" target="_blank">&copy; <b>Jawg</b>Maps</a> ' +
+					'<a href="https://jawg.io?utm_medium=leaflet-extra&utm_source=attribution" title="Tiles Courtesy of Jawg Maps" target="_blank">&copy; <b>Jawg</b>Maps</a> ' +
```
Comment (brunob) [inline leaflet-providers.js:371]: is this really useful ?

**Review by brunob — APPROVED** [review]

**Review by brunob — APPROVED** [review]

**Comment by brunob** [comment]
@jieter ?

**Comment by jieter** [comment]
Totally missed this one, sorry.

**Comment by jieter** [comment]
@clementgayvallet thanks!

**Pull request merged**